bmap-tools: 3.4 -> 3.6

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for bmap-tools is: "bmap-related tools"

meta.homepage for bmap-tools is: "https://github.com/intel/bmap-tools"

meta.changelog for bmap-tools is: ""


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes

- [Release on GitHub](https://github.com/intel/bmap-tools/releases/tag/v3.6)

- [Compare changes on GitHub](https://github.com/intel/bmap-tools/compare/v3.4...v3.6)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 3.6 with grep in /nix/store/px8wbf36ar1nxkmsj86w48xyd9vgwa2c-bmap-tools-3.6
- directory tree listing: https://gist.github.com/be126f5a26bbebfd8f795775ae5d6df8

- du listing: https://gist.github.com/ceb0011d8ea9fae86f19a84c7f054a9e

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
3 total rebuild path(s)

1 package rebuild(s)

1 x86_64-linux rebuild(s)
1 i686-linux rebuild(s)
0 x86_64-darwin rebuild(s)
1 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
bmap-tools
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/px8wbf36ar1nxkmsj86w48xyd9vgwa2c-bmap-tools-3.6 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A bmap-tools https://github.com/r-ryantm/nixpkgs/archive/80c4f5870863b59075a19597cf182fa1ce0216c9.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/px8wbf36ar1nxkmsj86w48xyd9vgwa2c-bmap-tools-3.6
ls -la /nix/store/px8wbf36ar1nxkmsj86w48xyd9vgwa2c-bmap-tools-3.6/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>bmap-tools</li>
  </ul>
</details>

---

###### Maintainer pings

cc @dezgeg for testing.

r-ryantm committed:
bmap-tools: 3.4 -> 3.6

danieldk reviewed (changes requested):
Fails with

```
$ bmaptool 
Traceback (most recent call last):
  File "/nix/store/px8wbf36ar1nxkmsj86w48xyd9vgwa2c-bmap-tools-3.6/bin/.bmaptool-wrapped", line 6, in <module>
    from bmaptools.CLI import main
  File "/nix/store/px8wbf36ar1nxkmsj86w48xyd9vgwa2c-bmap-tools-3.6/lib/python2.7/site-packages/bmaptools/CLI.py", line 42, in <module>
    from bmaptools import BmapCreate, BmapCopy, BmapHelpers, TransRead
  File "/nix/store/px8wbf36ar1nxkmsj86w48xyd9vgwa2c-bmap-tools-3.6/lib/python2.7/site-packages/bmaptools/BmapCopy.py", line 65, in <module>
    from six import reraise
ImportError: No module named six
```

danieldk committed:
bmap-tools: switch to Python 3

Python 3 has been supported by bmap-tools for several years:

https://github.com/intel/bmap-tools/commit/8ad1608ebc7ec4512e5e47cc81ace440ce5a2a15

danieldk committed:
bmap-tools: add six dependency

danieldk commented:
Fixed by switching to Python 3 and adding `six` as a dependency.

danieldk reviewed (approved):
LGTM and tested.

Result of `nixpkgs-review pr 113254` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>bmap-tools</li>
  </ul>
</details>
